docroot/themes/custom/yukonca_glider/yukonca_glider.theme

```diff
@@ -269,6 +269,11 @@ function yukonca_glider_preprocess_node__documents(&$variables) {
  */
 function yukonca_glider_preprocess_page(&$variables) {
   if (!empty($variables['node'])) {
+
+    if ($variables['node']->bundle() == 'topics_page') {
```

Comment:
You expect to have "basic_page" in [page.html.twig]. But there is no logic to set it.
Fix this please.

Reply:
Please review again.